doc/book/src/user/installation.md

```diff
@@ -0,0 +1,84 @@
+# Installation
+
+## Debian / Ubuntu
+
+The Electric Coin Company operates a package repository for 64-bit Debian-based distributions.
+
+### Fetching a New Signing Key
+
+First install the following dependency so you can talk to our repository using HTTPS:
+
+```
+sudo apt-get update && sudo apt-get install apt-transport-https wget gnupg2
+```
+
+Next add the Zcash master signing key to apt’s trusted keyring:
+
+```
+wget -qO - https://apt.z.cash/zcash.asc | gpg --import
+gpg --export B1C9095EAA1848DBB54D9DDA1D05FDC66B372CFE | sudo apt-key add -
+```
+
+### Adding and Installing from apt.z.cash
+
+Add the repository to your Bullseye sources:
+
+```
+echo "deb [arch=amd64] https://apt.z.cash/ bullseye main" | sudo tee /etc/apt/sources.list.d/zcash.list
+```
+
+Update the cache of sources and install Zcash:
+
+```
+sudo apt-get update && sudo apt-get install zcash
+```
+
+# Troubleshooting
+
+## Different Instruction Sets
+
+Only x86-64 processors are supported.
+
+## Missing sudo
+
+If you’re starting from a new virtual machine, sudo may not come installed. See this issue for instructions to get up and running: https://github.com/zcash/zcash/issues/1844
+
+## libgomp1 or libstdc++6 version problems
```

Comment:
libgomp1 was related to OpenMP which we no longer use. Also we don't dynamically link to libstdc++6 any more, we statically link libc++. This whole section should be removed.